geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/data/AddAttributeIndexCommand.scala

```diff
@@ -21,6 +20,24 @@ class AddAttributeIndexCommand extends AccumuloDataStoreCommand {
   override val params = new AddAttributeIndexParams
 
   override def execute(): Unit = {
+    new AddAttributeIndexCommandExecutor(params).run()
```

Comment:
could you add a comment on this and the other classes we had to break out as to why we did it (i.e. classpath issues)?